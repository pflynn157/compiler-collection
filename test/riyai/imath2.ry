func main -> i32 is
    var x1 : i32 := 10;
    var x2 : i32 := 5;
    print(x1);
    print(x2);
    
    print(x1 + 2);
    print(x1 - 2);
    print(x1 * 2);
    print(x1 / 2);
    print(x1 % 2);
    print(x1 % 3);
    print(x2 & 4);
    print(x2 | 4);
    print(x2 ^ 4);
    return 0;
end

