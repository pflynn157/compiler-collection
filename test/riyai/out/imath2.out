10
5
12
8
20
5
0
1
4
5
1
